tools/generator/repos/workTree.go

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package repos
```

Comment:
We have a similar package `tools/internal/repo` - investigate possibility to merge these two

Reply:
merged to tools/internal/repo